app/scripts/components/exploration/index.tsx

```diff
@@ -1,142 +1,89 @@
-import React, { useEffect, useState } from 'react';
-import { Panel, PanelGroup, PanelResizeHandle } from 'react-resizable-panels';
-import styled from 'styled-components';
-import { themeVal } from '@devseed-ui/theme-provider';
-import { TourProvider } from '@reactour/tour';
-
+import React, { useState } from 'react';
+import { Link } from 'react-router-dom';
+import { DevTools } from 'jotai-devtools';
 import { useAtom, useSetAtom } from 'jotai';
-import Timeline from './components/timeline/timeline';
-import { ExplorationMap } from './components/map';
-import { useAnalysisController } from './hooks/use-analysis-data-request';
-import { PopoverTourComponent, TourManager } from './tour-manager';
-import { TimelineDataset } from './types.d.ts';
-import { selectedCompareDateAtom, selectedDateAtom } from './atoms/dates';
-import { CLEAR_LOCATION, urlAtom } from '$utils/params-location-atom/url';
-import { legacyGlobalStyleCSSBlock } from '$styles/legacy-global-styles';
-
-// @TODO: "height: 100%" Added for exploration container to show correctly in NextJs instance but investigate why this is needed and possibly work to remove
-const Container = styled.div`
-  display: flex;
-  flex-flow: column;
-  flex-grow: 1;
-  height: 100%;
-
-  .panel-wrapper {
-    flex-grow: 1;
-  }
-
-  .panel {
-    display: flex;
-    flex-direction: column;
-    position: relative;
-  }
-  * {
-    ${legacyGlobalStyleCSSBlock}
-  }
-  .panel-timeline {
-    box-shadow: 0 -1px 0 0 ${themeVal('color.base-100')};
-  }
-
-  .resize-handle {
-    flex: 0;
-    position: relative;
-    outline: none;
-    display: flex;
-    align-items: center;
-    justify-content: center;
-    width: 5rem;
-    margin: 0 auto -1.25rem auto;
-    padding: 0rem 0 0.25rem;
-    z-index: 1;
 
-    ::before {
-      content: '';
-      display: block;
-      width: 2rem;
-      background: ${themeVal('color.base-200')};
-      height: 0.25rem;
-      border-radius: ${themeVal('shape.ellipsoid')};
-    }
-  }
-`;
+import { DatasetSelectorModal } from '$components/exploration/components/dataset-selector-modal';
```

Comment:
This file is largely the same as the prior container  but I updated all the relative path imports.